main.c

```diff
@@ -0,0 +1,142 @@
+//Why the fuck did they fuck with a good thing? Now I gotta download a buncha shit to use github? fuck it.
+//This is a simple homemade game of rock paper scissors
+//It uses a digispark running at 16.5MHz on the microneucleus bootloader
+//It is glitchy to upload. You'll need a USB 2.0 hub, new 3.0 hubs haven't worked for me.
+
+// constants won't change. They're used here to
+// set pin numbers and values for musical tones
+
+const int c   = 261;
+const int d   = 294;
+const int e   = 329;
+const int f   = 349;
+const int g   = 391;
+const int gS  = 415;
+const int a   = 440;
+const int aS  = 455;
+const int b   = 466;
+const int cH  = 523;
+const int cSH = 554;
+const int dH  = 587;
+const int dSH = 622;
+const int eH  = 659;
+const int fH  = 698;
+const int fSH = 740;
+const int gH  = 784;
+const int gSH = 830;
+const int aH  = 880;
+
+const int speakerPin = 0;	// PWM pin for speaker on the digispark it's pin P0
+const int buttonPin  = 1;	// the number of the button pin P1
+const int rockPin    = 2;	// the number of the ROCK LED pin P2
+const int paperPin   = 3;	// the number of the PAPER LED pin P3
+const int scissorPin = 4;	// the number of the SCISSOR LED pin P4
+
+int buttoncounter = 0;		// integer value for storing how many times the game start reed switch has been actuated
+int counter       = 0;		// integer value for storing the counter for making muzak
```

Comment:
Understandable. I just wanted to pretend my computer science degree is worth something for once. Nice work on the PR, this is a huge improvement and beautiful code. @Bumblefuck :shipit: